shared/src/native-src/pal.h

```diff
@@ -39,7 +39,38 @@ struct has_deprecated_log_folder<T, decltype(T::logging_environment::deprecated_
 };
 
 template <class TLoggerPolicy>
-inline shared::WSTRING GetDatadogLogFilePath(const std::string& file_name_suffix)
+inline fs::path GetDefaultLogDir()
+{
+    bool isAas = false;
+    TryParseBooleanEnvironmentValue(GetEnvironmentValue(WStr("DD_AZURE_APP_SERVICES")), isAas);
+
+    if (isAas)
+    {
+#ifdef _WINDOWS
+        return WStr("C:\\home\\LogFiles\\datadog\\dotnet\\");
+#else
+        return WStr("/home/datadog/dotnet");
+#endif
+    }
```

Comment:
@andrewlock I would like your input on that.
I told to serverless team and they recommended this